src/ansys/fluent/core/graphics_definitions/matplotlib/plotter_defns.py

```diff
@@ -0,0 +1,223 @@
+"""Module providing matplotlib plotter functionality."""
+
+from typing import List, Optional
+
+import matplotlib.pyplot as plt
```

Comment:
This introduces matplotlib "dependency" in core. We are bringing some shared object definitions from visualization package to here so that we can use them in ui. Is there code in this module that needs to be used outside of visualization. If so, it implies that we need to break this module up and keep the matplotlib-dependent stuff in visualization.